### PR TITLE
few Python 3 related fixes

### PR DESCRIPTION
**Pull request by sa2ajj**
* do not use 'print' use 'twisted.python.log'
* use 'except ... as ...' instead of 'except ..., ...'

### CODE REVIEW

File: master/buildbot/www/oauth2.py [hunk 1]
```diff
@@ -51,7 +54,7 @@ def renderLogin(self, request):
                 details.update(infos)
             request.getSession().user_info = details
             state = request.args.get("state", [""])[0]
-            print repr(state)
+            log.msg(repr(state))
```
Comment (tardyp) [inline master/buildbot/www/oauth2.py]: I think that one is a left-over debug print.

File: master/buildbot/www/oauth2.py [hunk 2]
```diff
@@ -15,11 +15,14 @@
 from posixpath import join
 
 import requests
+
```
Comment (tardyp) [inline master/buildbot/www/oauth2.py]: isort should group those 2 paragraphs back

Comment (sa2ajj) [inline master/buildbot/www/oauth2.py]: yup: build failure :smile:

File: master/buildbot/www/oauth2.py (hunk 1 above)
Comment (sa2ajj) [inline master/buildbot/www/oauth2.py]: it might be, however since it's in the code, i'd like to make it Python 3 compatible.

Comment (tardyp) [inline master/buildbot/www/oauth2.py]: or just delete the line

**Comment by sa2ajj** [comment]
Yeah, deleting those lines is probably a better idea.

**Comment by tardyp** [comment]
thanks

**Pull request merged**